integreat_cms/cms/templates/pois/_map_preview_overlay.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+
+<div id="map_preview_overlay" class="fixed items-center justify-center overflow-scroll inset-0 bg-opacity-75 bg-gray-800 z-50 cursor-pointer hidden">
+    <div class="cursor-auto flex flex-col justify-center max-h-fit w-[850px] max-w-full w-auto px-[10px] z-50 m-auto">
+        <div class="bg-white opacity-100 content rounded shadow-md w-full">
+            <button type="button" id="btn-close-preview" class="float-right m-4 p-1 rounded-full hover:bg-blue-500 hover:text-white">
+                <i icon-name="x-circle" class="h-8 w-8"></i>
+            </button> 
+            
+            <div id='map' style='width: 400px; height: 300px;'>
+                <div id="no_coordinates_alert" class ="hidden">
+                    {% trans 'Map preview will be available when coordinates are given.' %}
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
Do you think you could render this directly in the sidebar below "Position"?

Reply:
Do you mean the map preview shall be an embedded part of the form, not a pop-up?

Reply:
I would prefer that, yes. Maybe with an additional button to show a larger map which opens the pop-up, but it would be cool if a small map would directly show the position at the first glance.

Reply:
Ok👍